Alexis-Vacilli/myBrand-api
# Ft signup

#### What does this PR do?
[what does this PR do?]
> This is the PR for creating the user
#### Description of Task to be completed
[the tasks that you completed on the feature]
> The task completed was about creating the user and then run write tests for that endpoint 
#### How should this be manually tested?
> Clone this repo https://github.com/Alexis-Vacilli/myBrand-api.git  
npm install: To install all the dependencies  
npm run cover: To run tests and see the coverage
#### Any background context you want to provide?
> None 
#### What are the relevant Trello story?
> https://trello.com/c/Bx0f4M4d
#### Screenshots (if appropriate)
>
